panther_analysis_tool/main.py

```diff
@@ -1294,6 +1302,15 @@ def setup_parser() -> argparse.ArgumentParser:
         "type": strtobool,
         "help": "Meant for advanced users; allows skipping of extra keys from schema validation.",
     }
+    available_destination_name = "--available-destination"
```

Comment:
Let me know if I should add documentation for the new CLI parameter!

Reply:
I think repeating the CLI parameter is considered a safer approach to handle arbitrary input, because choosing a delimiter always complicates the parsing process (or the user must apply some custom character escaping). I think we currently allow the comma character in a display name, but since we already use comma-delimited input in other parameters, I'll change it to your suggestion.

Reply:
I see, I didn't think about the fact that destinations may have commas in the name. Feel free to leave it as is then, otherwise we'll likely just have to revert it the first time we get a customer that puts a comma in the name.

Reply:
Great, I'll just leave it as is then for now. I also verified by adding a destination from the UI and we do allow comma.